packages/vendure-plugin-customer-managed-groups/src/api-extensions.ts

```diff
@@ -35,18 +36,68 @@ export const shopSchema = gql`
     firstName: String
     lastName: String
     emailAddress: String
+    addresses: [CustomerManagedGroupAddressInput]
```

Comment:
Small improvement: 
```suggestion
    addresses: [CustomerManagedGroupAddressInput!]
```
`[CustomerManagedGroupAddressInput!]`  means `addresses` is optional, but if you supply this argument, it can not be null: `addresses: [ { // address input } ]` is allowed, but `addresses: [null]` is not

`[CustomerManagedGroupAddressInput]`  means that `adresses: [null]` is also allowed, but that's usually not what we want.